Utilities/bootstrap

```diff
@@ -179,24 +180,65 @@ class Target(object):
             str(bool(self.is_library)).lower()), file=output)
         print(file=output)
 
+def parse_manifest():
+    targets = []
+
+    with open(os.path.join(g_project_root, "Package.swift")) as f:
+        # find first targets section
+        while re.match(r'\s*targets: ', next(f)) == None:
+            continue
+
+        for line in f:
+            if re.match(r'\s*\]\)', line):  #TODO this is why we should do proper tokenization
+                break
+        
+            if re.match(r'\s*Target\(', line):
```

Comment:
If we are going to do this, I would propose normalizing the Package.swift we have to be super-regular, so that the parsing logic can be very very strict. So for example if we required 

```
/* BEGIN-TARGETS */
...
  Target(
     /* Comment */
     name: NAME,
     dependencies: [DEPS (even if empty)]
...
/* END-TARGETS */
```
then the parsing logic can become something like (no error handling):

```
data = open("Package.swift").read()

# Split on the begin/end target markets.
targetData, = re.match(r'/*BEGIN-TARGETS*/(.*)/*END-TARGETS*/', re.DOTALL|re.MULTILINE).groups()

# Iterate over each target -- we expect a very strict structure.
#
# FIXME: My re is no doubt wrong.
for (targetName, targetDeps) in re.finditer(targetData, r"Target\(\s*(/*.*?*/)?name: (.*?),\s*dependencies: (\[.*?\])\)", re.DOTALL|re.MULTILINE):
  # Cheat to get the deps list, because Python == Swift on the supported subset.
  targetDeps = eval(targetDeps)
  ... process this target and its deps ...

```